documentation/testing.html

```diff
@@ -208,3 +184,28 @@ <h3 id="using-configuration-builders">Using configuration builders</h3>
 
 }
 </pre>
+
+
+
+<h2 id="system-tests">System tests</h2>
+<p>
+A System Test is a test that tests a full Vespa application, using its endpoints.
+A System Test is invoked before a change to the production environment:
+<ol>
+  <li>Application package deployment</li>
+  <li>Vespa software upgrade</li>
+</ol>
+Vespa System Tests use <a href="https://junit.org/junit5/">JUnit 5</a> annotations.
+Tests are annotated with <code>@Test</code> as unit tests.
+The test class must be annotated with <code>@Tag("SystemTest")</code>
```

Comment:
Needs to be `ai.vespa.hosted.cd.SystemTest`, which is held in the static `ai.vespa.tenant-cd.SystemTest.name` field.

Reply:
This can be changed later, though.

Reply:
ok, but why ai.vespa.hosted.cd.SystemTest?  this is just instrumentation to maven for which tests to run - or is there more to it?

Reply:
Right now that's the required name (comes from the @Category, which needs a class argument). We can change it, or allow both. If it doesn't need to work today, and I guess it doesn't, then "SystemTest" is fine.